pkg/kubelet/apis/config/types.go

```diff
@@ -389,6 +389,11 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration
+	// PodPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based priorities. Requires the GracefulNodeShutdown feature gate to be enabled.
```

Comment:
specifies the shutdown grace period for Pods based on their associated priority class value.